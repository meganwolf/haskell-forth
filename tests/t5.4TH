25.0 10.0 * 50.0 + .
25.0 10.0 -  CR .
5.0 2.0 / CR .
5.0 2.0 ^ CR .
5.0 3.0 ^ 8.0 + CR .
