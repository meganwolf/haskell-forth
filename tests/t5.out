300.0
15.0
2.5
25.0
133.0